src/app/view/components/dot-persona-selector/dot-persona-selector.component.scss

```diff
@@ -0,0 +1,70 @@
+@import "variables";
+@import "mixins";
+
+.dot-persona-selector {
+    .ui-overlaypanel-content {
+        padding: $basic-padding * 2 0;
+    
+        .searchable-dropdown__search {
+            margin: 0 $basic-padding * 2;
+    
+            .searchable-dropdown__search-icon {
+                right: $basic-padding;
+            }
+    
+            .searchable-dropdown__search-inputfield {
+                @include form-field-border;
+                padding: 0 $basic-padding * 4 0 $basic-padding * 2;
+            }
+        }
+    }
+}
+
+.dot-persona-selector__container {
+    align-items: center;
+    cursor: pointer;
+    display: flex;
+    height: 48px;
+    position: relative;
+    width: 206px;
+
+    dot-avatar {
+        margin-right: $basic-padding;
+    }
+
+    .dot-persona-selector__label-container{
+        overflow: hidden;
+        text-overflow: ellipsis;
+        white-space: nowrap;
+
+        span {
+            display: block;
+        }
+        .dot-persona-selector__label {
+            color: $gray;
+            flex: 1;
+            font-size: $font-size-small;
+            line-height: $basic-padding * 2;
+        }
+        .dot-persona-selector__name {
+            color: $black;
+            font-weight: $semi-bold;
+            line-height: $basic-padding * 2;
+            margin-right: $basic-padding * 3;
+            overflow: hidden;
+            text-overflow: ellipsis;
+            white-space: nowrap;
+        }
+    }
+
+    dot-icon {
+        bottom: $basic-padding / 2;
+        position: absolute;
+        right: 0;
+
+        i.material-icons {
+            color: $gray;
+            font-size: 24px;
```

Comment:
We're going to need font size variables for icons that we can reuse.

Reply:
$md-icon-size-big